backend/app/model/location.rb

```diff
@@ -22,6 +22,7 @@ class Location < Sequel::Model(:location)
                     :contains_records_of_type => :location_function,
                     :corresponding_to_association  => :location_function)
 
+  
```

Comment:
This barely matters, but maybe get rid of this extra line just because there are no other changes to this file in the PR.

Reply:
I would but it seems harder to remove this from the PR :)